go.mod

```diff
@@ -75,9 +75,9 @@ require (
 	google.golang.org/genproto v0.0.0-20211208223120-3a66f561d7aa
 	google.golang.org/grpc v1.43.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.22.4
```

Comment:
We had a lot of issues with updating this.  I believe we have to upgrade to go 1.18 in order to actually get the tests to run.  

We have dependabot PR that are trying to update this but they keep failing tests.  

I'd ask that if you need to update this you should do it in a separate PR.

Reply:
Actually I need to import a new library kubernetes/client-go, and it updated a lot of dependencies, need to figure out an appropriate version

Reply:
I think that was already present in the go.mod.